src/constants.js

```diff
@@ -26,11 +26,12 @@ module.exports = {
   },
 
   Phases: {
-    bet: 'bet',
-    vote: 'vote',
-    setResult: 'setResult',
-    finalize: 'finalize',
-    withdraw: 'withdraw',
+    BETTING: 'betting',
```

Comment:
can we make these lowercase? Sometimes I feel like we're switching between uppercase, lowercase, pascal case, and camelCase and it's nice to just have the keys of the maps be the same.
For instance:
if I want to do this
```js
const phase = getPhase(oracle);
const appLocation = AppLocation[phase]
```
Or vice versa. I can't really do either because all our keys are different amongst our constants. It would be great if we could make the keys the same so we could reuse them in various parts of the app yeah?
```js
Phases: {
  betting: 'betting'
  ...
},
AppLocation: {
  betting: '/bet'
  ...
}
```
Also, I had `betting` etc, but switched to `bet` for brevity. So chill with either on that front :)

Reply:
Does what I'm suggesting make sense? I can provide other examples if it doesn't :)

Reply:
how do you feel about ALL UPPERCASE for these constants? that's generally a recognized standard cross-language convention.

Reply:
I personally use UPPERCASE for this kind of constants as well.  but if so, shall we refactor whole `constants.js` and all related files in this PR?

Reply:
I have updated the phases to all uppercase.

@amazingandyyy I will make a separate PR for refactoring the rest of this. When doing PRs, let's try to keep the scope of it as on point as possible.